code/modules/organs/subtypes/robotic.dm

```diff
@@ -11,6 +11,14 @@
 	var/list/forced_children = null
 	var/attack = 0 			//Attack mode
```

Comment:
If variable must contain bool value, use TRUE / FALSE, if variable must contain object, use null to clear and specify variable type